src/DummyTest.java

```diff
@@ -1,9 +1,154 @@
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.function.Executable;
+
+
 
 import static org.junit.jupiter.api.Assertions.*;
 
 // FIXME: remove this class
 public class DummyTest {
+
+       
+    @Test
+    @DisplayName("Test in Entity")
+    void testInConstructor() {
```

Comment:
```suggestion
    public void testInConstructor() {
```

És a többinél is ugyanígy.